src/containers/MainContainer.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import ShortForm from '../components/short-form/ShortForm';
+import Display from '../components/display/Display';
+import List from '../components/list/List';
+import { getUserInfo, getUserRepos } from '../services/githubAPI';
+
+export default class MainContainer extends Component {
+
+    state = {
+      userName: '',
+      userInfo: {},
+      repos: []
+    }
+
+    handleChange = ({ target }) => {
+      this.setState({ userName: target.value });
+    }
+
+    handleSubmit = event => {
+      event.preventDefault();
+
+      const { userName } = this.state;
+      Promise.all([
+        getUserInfo(userName),
+        getUserRepos(userName) 
+      ])
+        .then(([userInfo, repos]) => this.setState({ userInfo, repos }));
+    }
+
+    render() {
+      const { repos, userName, userInfo: { login, followers, following, html_url } } = this.state;
+      return (
+        <>
+          <ShortForm userName={userName} onChange={this.handleChange} onSubmit={this.handleSubmit} />
+          <Display name={login || ''} followerCount={followers || 0} followingCount={following || 0} githubLink={html_url || ''} />
```

Comment:
You can do this `login || ''` in you state up top, by giving `userInfo` these properties with empty strings. It will make you code look much cleaner.